Dockerfile.dapper

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && \
+    apt-get install -y wget build-essential python2.7 python2.7-dev python-virtualenv vim && \
+    rm -f /bin/sh && ln -s /bin/bash /bin/sh
+
+ENV DAPPER_SOURCE /go/src/github.com/longhorn/longhorn-tests
+
+WORKDIR ${DAPPER_SOURCE}
+
+ENV VIRTUAL_ENV=${DAPPER_SOURCE}/manager/integration/.venv
+
+RUN python2 -m virtualenv --python=/usr/bin/python2 ${VIRTUAL_ENV}
+
+CMD ["./manager/integration/scripts/flake8-check.sh"]
```

Comment:
```
/usr/bin/docker: Error response from daemon: OCI runtime create failed: container_linux.go:345: starting container process caused "exec: \"./manager/integration/scripts/flake8-check.sh\": permission denied": unknown.
```

Seems flake8-check.sh is not executable.

Reply:
`flake8-check.sh` is now executable